vuepress/docs/next/docs/consume/entando-apis.md

```diff
@@ -3,139 +3,80 @@ sidebarDepth: 2
-```
-kubectl -n entando edit deployment/quickstart-server-deployment
-```
+## Models
+All of the model classes returned by the Entando core are annotated so that the model definition is included in the Swagger documentation. At the bottom of the Swagger page all of the model classes returned by the API endpoints can be found.
 
-2. (Optional) Scale the deployment `spec.replicas` to 0 before updating the deployment. This is necessary if you're using an in-memory database, e.g. the default quickstart configuration, and will prevent database errors that can happen on an immediate restart after the profile is changed. Save the deployment to apply the change. 
+## Enable the Swagger UI
 
-3. Find the entando-de-app env variables section under `spec.template.spec.containers.env[image: entando-de-app]`
+The Swagger UI can be enabled or disabled in a running Entando instance by modifying the `SPRING_PROFILES_ACTIVE` environment variable for the `entando-de-app` container. 
 
-4a. To enable the swagger UI, add the SPRING_PROFILES_ACTIVE environment variable, if it's missing, or add `swagger` to its comma-delimited list.
+1. Edit the `entando-de-app` deployment. The deployment and namespace may have different names outside of a quickstart environment. 
 
+>Use the [ent CLI](../getting-started/entando-cli.md) to send commands to Kubernetes from the host machine.
 ```
-        - name: SPRING_PROFILES_ACTIVE
-          value: default,swagger
+kubectl -n entando edit deployment/quickstart-deployment
 ```
-4b. To disable the swagger UI, remove `swagger` from the value.
-
-5. (Optional) Reset the deployment `spec.replicas` back to 1.
-
-6. Save the deployment to apply the change. 
-
-## How to find your client secret
-You'll need your client credentials to execute the Entando APIs. 
-
-1. Login into your Keycloak instance
-
-2. Go to `Administration → Clients`
-
-3. Select the desired client (e.g. in a quickstart environment this is `quickstart-server`)
-
-4. Click on the `Credentials` tab to get the secret 
-
-## Setup in local environment
-
-You may prefer to run a local standalone Entando application for some tasks. You'll need Java 11, maven, and Keycloak for authentication. See [these instructions](https://github.com/entando/app-builder/blob/master/with-keycloak.md) to setup a standalone Keycloak.
-
-### Configure Keycloak
-
-Configure your Keycloak client in order to support Swagger UI. A quickstart environment has this pre-configured.
 
-1. Login to your Keycloak instance
+2. (Optional) Scale the deployment `spec.replicas` to 0 before updating the deployment. This is necessary if you're using an in-memory database, e.g. the default quickstart configuration, and will prevent database errors that can happen on an immediate restart after the profile is changed. Save the deployment to update. 
 
-2. Access the Administration console
+3. In the deployment YAML, find the env variables section under `spec.template.spec.containers.env[image: entando-de-app]`
```

Comment:
either spell out the abbreviation "environment variables" or format "env variables" with backticks, etc., so the reader knows this is the actual name of the section and not a casual instruction